anitya/lib/backends/rubygems.py

```diff
@@ -74,3 +74,27 @@ def get_versions(cls, project):
                 'Project or version unknown at %s' % url)
 
         return [data['version']]
+
+    @classmethod
+    def check_feed(cls):
+        ''' Return a generator over the latest 50 uploads to rubygems.org
+
```

Comment:
Will this be checked often enough to be sure that we don't miss anything? Should I open upstream RFE asking to implement some dynamic filter, e.g. "give all updates since gem foo v1.2.3 was released" or alternatively "give me all project since some timestamp"?